web/src/hotkey.js

```diff
@@ -64,6 +65,10 @@ import {user_settings} from "./user_settings";
 import * as user_topics_ui from "./user_topics_ui";
 
 function do_narrow_action(action) {
+    if (message_lists.current === undefined) {
+        return false;
+    }
```

Comment:
This seems like it could be assert for all the cases but this, the `p` shortcut:

```
        case "narrow_private":                                                                          
            return do_narrow_action((_target, opts) => {                                                
                narrow.by("is", "dm", opts);                                                            
            });                                                                                         
```

I think this might break that shortcut when looking at inbox/recent.

Reply:
Yeah, it's broken. Changed `narrow_private` to directly use
`narrow.by("is", "dm", {trigger: "hotkey"});`.